packages/app-builder-lib/templates/nsis/include/FileAssociation.nsh

```diff
@@ -103,10 +97,7 @@
 
 
 !macro APP_UNASSOCIATE EXT FILECLASS
-  ; Backup the previously associated file class
-  ReadRegStr $R0 SHELL_CONTEXT "Software\Classes\.${EXT}" `${FILECLASS}_backup`
-  WriteRegStr SHELL_CONTEXT "Software\Classes\.${EXT}" "" "$R0"
-
+  DeleteRegValue SHELL_CONTEXT "Software\Classes\.${EXT}\OpenWithProgids" "${FILECLASS}"
```

Comment:
Please refer to the following document for the behavior when uninstalling.
https://learn.microsoft.com/windows/win32/shell/fa-file-types#deleting-registry-information-during-uninstallation

---

Maybe there are applications like electron-builder that write the ProgID to the default key only.
If you are considerate of such applications, it may be better not to write anything to the default key.
(But doing so would change previous behavior.)